content/en/docs/Tasks/update-and-ship-operator.md

```diff
@@ -0,0 +1,192 @@
+
+
+This graph maintains that:
+
+- Any Operator found in Old CatalogSource has a single replacement in New CatalogSource.
+
+- Any Operator found in New CatalogSource has a single replacement in New CatalogSource.
+
+- If the bad update has not yet been installed, it will never be.
+
+## Replacing Multiple Operators
+
+Creating the New CatalogSource as described requires publishing CSVs that replace one Operator, but can `skip` several. This can be accomplished using the `skipRange` annotation:
+
+        olm.skipRange: <semver_range>
+
+where `<semver_range>` has the version range format supported by the [semver library](https://github.com/blang/semver#ranges).
+
+When searching catalogs for updates, if the head of a channel has a `skipRange` annotation and the currently installed Operator has a version field that falls in the range, OLM updates to the latest entry in the channel.
+
+The order of precedence is:
+
+- Channel head in the source specified by sourceName on the Subscription, if the other criteria for skipping are met.
+
+- The next Operator that replaces the current one, in the source specified by sourceName.
+
+- Channel head in another source that is visible to the Subscription, if the other criteria for skipping are met.
+
+- The next Operator that replaces the current one in any source visible to the Subscription.
+
+`skipRange example:`
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+## Z-stream support
+
+A z-stream (patch release) needs to replace all previous z-stream releases for the same minor version. OLM doesn't care about major/minor/patch versions, we just need to build the correct graph in a catalog.
+
+In other words, we need to be able to take a graph as in "Old Catalog" and, similar to before, generate a graph as in "New Catalog"
+
+![replace Image](/docs/Tasks/images/replace.png)
+
+`Figure 5. Replacing several Operators`
+
+This graph maintains that:
+
+- Any Operator found in Old CatalogSource has a single replacement in New CatalogSource.
+
+- Any Operator found in New CatalogSource has a single replacement in New CatalogSource.
+
+- Any z-stream release in Old CatalogSource will update to the latest z-stream release in New CatalogSource.
+
+- Unavailable releases can be considered "virtual" graph nodes; their content does not need to exist, the registry just needs to respond as if the graph looks like this.
+
+# Ship Operator
+
+For the shimpment of an operator, first make sure that manifests gets created with the updated version of operator. Check below link for mre details on operator manifest creation process.
+
+[How to create Operator manaifest?](/docs/tasks/creating-operator-manifests/)
```

Comment:
The links should follow the standard. See that we should: 
- [text](alias)

And then, at the bottom of the page have the link alias such as; https://raw.githubusercontent.com/operator-framework/olm-docs/master/content/en/docs/best-practices/common.md

Reply:
Modified according to suggestion.